runtime/proto/service/log_replication.proto

```diff
@@ -52,4 +52,19 @@ enum LogReplicationEntryType {
   SNAPSHOT_REPLICATED = 4;
   SNAPSHOT_END = 5;
   SNAPSHOT_TRANSFER_COMPLETE = 6;
+}
+
+message ReplicationSessionMsg {
+  string remoteClusterId = 1;
+  string localClusterId = 2;
+  string client = 3;
+  // TODO: this can be replaced with the replication tag
+  ReplicationModel replicationModel = 4;
+}
```

Comment:
I might missed some context, also saw this change in Anny's branch for session manager. So "LogReplicationSession" will no longer be some objects holding all the FSM/Reader/Writer for its session, but a kind of identifier defined by protobuf? and embedded into header msg?

Reply:
haven't finished review to get more context, but seems like this protobuf msg is already equivalent to what we have in `ReplicationSession.java`

Reply:
yes, Anny and my changes have some overlap where session is involved. Anny's has it thorough, the ReplicationSession class has been replaced with a proto msg, so I'll be rebasing on top of that PR once thats available.